bubble_sort.rb

```diff
@@ -0,0 +1,36 @@
+def bubble_sort(arr)
+  test = false
+  until test
+    test = true
+    arr.each_with_index do |_a, index|
+      next if index == arr.length - 1
+
+      if arr[index] > arr[index + 1]
+        arr[index], arr[index + 1] = arr[index + 1], arr[index]
+        test = false
+      end
+    end
+  end
+
+  arr
+end
+arr = [4, 3, 78, 2, 0, 2]
+p bubble_sort(arr) # = [0, 2, 2, 3, 4, 78]
```

Comment:
Delete commented code form your code. Also, as mentioned earlier by the first reviewer, remove code used for validating the output, leave only the methods.